renovate.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Those look fair to start. And yea guess that would be needed. I'll put back in draft till them

Reply:
I'm proposing a new approach this time around: don't automatically generate beachball change files. Instead it's up to code owners to add them manually (I'm willing to be the main person that does this). I've reduced the frequency of updates to be monthly instead of weekly to lessen the burden.

Some rationale:
1. the automatic changefile generation in SystemLinkShared has been sub-optimal - pushing to the branch causes Renovate to stop updating PRs and the change files don't contain much useful info.
2. bringing the same infrastructure to Nimble would take work to figure out the right git commands / permissions in GitHub Actions rather than AzDO. I'd rather not block Renovate on that work since we're starting to get nudges from the rich text folks to have a less manual approach for staying on top of vulnerabilities.
3. even if we're too lazy to complete these Renovate PRs in a timely manner, we can still treat them as "canary" builds to let us know when new dependency versions might break us

Reply:
> we're starting to get nudges from the rich text folks to have a less manual approach for staying on top of vulnerabilities.

So they use a version range specifier that allows all minor versions (as they should):

https://github.com/ni/nimble/blob/6bb30f55e68c85eac7225ee430920b0729c3feeb/packages/nimble-components/package.json#L67-L77

They shouldn't be blocked from uptaking new minor versions in apps. Just saying if it's valuable to them their apps could be updating those dependencies with `npm update` / renovate (I assume). It's of course nicer that the updates run through nimble's tests but not strictly required for them to adopt. A new app made from scratch or recreating the lock file would use the latest supported versions in the supported range.

This may be relevant if they push back on a monthly update cadence.

Reply:
This last bullet didn't even register before:

> even if we're too lazy to complete these Renovate PRs in a timely manner, we can still treat them as "canary" builds to let us know when new dependency versions might break us

But yea that's great! Had been thinking of setting up a separate pipeline for that. This should be even better since it'll be prepped for merging 👍